observer/bin/migrate.js

```diff
@@ -1,5 +1,3 @@
-import { DATABASE_URL } from '../lib/config.js'
-import { migrateWithPgConfig as migrateStatsDB } from '@filecoin-station/spark-stats-db-migrations'
+import { migrate } from '@filecoin-station/spark-stats-db'
 
-console.log('Migrating spark_stats database')
-await migrateStatsDB({ connectionString: DATABASE_URL })
+await migrate()
```

Comment:
I think now that the observer needs to access the spark-evaluate database, the migration CLI tool should migrate the evaluate database too. Otherwise the following workflow no longer works: 

```
npm run migrate -w observer && npm start -w observer
```

Reply:
it's migrating both 👍